src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss

```diff
@@ -27,15 +27,22 @@
   color: var(--ubs-primary-white);
   font-family: var(--primary-font);
   border: 1px solid var(--ubs-primary-light-grey);
-  display: flex;
-  justify-content: center;
+  width: 200px;
+  text-align: center;
   font-size: 14px;
   min-height: 39px;
   min-width: 48px;
   overflow: hidden;
   word-wrap: break-word;
   white-space: nowrap;
   overflow-wrap: break-word;
+  flex: none;
+  display: block;
+  line-height: 2.8;
+}
+
+.cdk-column-select {
+  width: 100px !important;
```

Comment:
Can we avoid !important here?
Maybe we can use :host or :ng-deep instead?
https://blog.angular-university.io/angular-host-context/

If not, let it be :)

Reply:
If not, let it be :)